tests/comfy_cli/command/generate/test_app.py

```
@@ -415,6 +415,151 @@ def get(self, *a, **kw):
+
+    def fake_post(url, *, json=None, headers=None, timeout=None, **_):
+        captured["body"] = json
+        return httpx.Response(200, json={"id": "job-xyz", "polling_url": "https://x/poll"})
+
+    monkeypatch.setattr(gen_app.client.httpx, "post", fake_post)
+    r = runner.invoke(
+        cli_app,
+        ["generate", "flux-kontext", "--prompt", "edit it", "--input_image", str(img), "--async"],
+    )
+    assert r.exit_code == 0, r.stdout
+    import base64 as _b64
```

Pylint (4.0.5)

[convention] Import outside toplevel (base64)

(C0415)